src/Hidden/Hidden.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import {useMedia} from '../hooks/useMedia'
+import {viewportRanges} from '../hooks/useResponsiveValue'
+
+type Viewports = 'narrow' | 'regular' | 'wide'
+
+type HiddenProps = {
+  on: Array<Viewports> | Viewports
+  children: React.ReactNode
+}
+
+export const Hidden = ({on: hiddenViewports, children}: HiddenProps) => {
```

Comment:
I loved the `when` convention from the docs, it was nice to read through it and think: "Okay, this is hidden when the viewport is narrow and wide". Is there a mental scheme you'd recommend with `on`?

Reply:
I kept going back and forth with `when` and `on` in my head. Sorry for the mix-up. Vini was also leaning towards `when`. Will change to that. 
For me its just a shorter sentence to say ` hidden on narrow viewport` vs `hidden when viewport is narrow`. When I think of it, I feel like its a non-native english speaker thing. 🤣